.github/workflows/reusable_build_images.yml

```
@@ -215,14 +215,18 @@ jobs:
         with:
           path: artifacts
 
+      - name: Pull latest providers
+        run: cargo run pull
```

Not sure, but you might need the separator there

```suggestion
        run: cargo run -- pull
```